src/System.Security.Principal.Windows/tests/NTAccount.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using Xunit;
+
+namespace System.Security.Principal.Windows.Tests
+{
+    public class NTAccountTest
+    {
+        [Fact(Skip = "This test needs a machine in a domain but off line.")]
+        public void Translate_Fail_Domain_Offline()
+        {
+            var nta = new NTAccount("foobar");
```

Comment:
Somewhere someone has a domain or user named foobar. Best to use something like `Guid.NewGuid().ToString("N")`